pandas/tests/groupby/test_groupby.py

```diff
@@ -2996,3 +2996,15 @@ def test_groupby_multi_index_codes():
 
     index = df_grouped.index
     tm.assert_index_equal(index, MultiIndex.from_frame(index.to_frame()))
+
+
+def test_groupby_keys_1length_list():
```

Comment:
Since this behavior is already hit in tests, I don't think we need to add a new test here.

Reply:
thanks, I removed the new test